speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -188,6 +250,61 @@ static void SaveSettings(void* /* dummy */)
     GfuiScreenActivate(s_nextHandle);
 }
 
+/// @brief                    Sets all the checkboxes and radiobuttons in the researcher menu
+void InitializeMenuButtons()
+{
+    GfuiCheckboxSetChecked(s_scrHandle, m_indicatorsGui[0], m_indicators.Audio);
+    GfuiCheckboxSetChecked(s_scrHandle, m_indicatorsGui[1], m_indicators.Icon);
+    GfuiCheckboxSetChecked(s_scrHandle, m_indicatorsGui[2], m_indicators.Audio);
```

Comment:
```suggestion
    GfuiCheckboxSetChecked(s_scrHandle, m_indicatorsGui[2], m_indicators.Text);
```
This should probably be text right?